clarity_epp/export/illumina.py

```diff
@@ -157,6 +198,12 @@ def get_project(projects, urgent=False):
         elif sample_header:  # Samples header seen, so continue with samples.
             data = line.rstrip().split(',')
 
+            # Fix sample name -> use sequence name
+            try:
+                data[sample_name_index] = sample_sequence_names[data[sample_name_index]]
+            except KeyError:
```

Comment:
when does a keyerror occur here, and when it happens, it can pass?
later in line 167/214 (below the next pass) you use
`data[sample_id_index] = data[sample_name_index]`

so if `data[sample_name_index]` creates a keyerror, the line above will also crash?

Reply:
A good catch, the key error comes from `sample_sequence_names`, this only contains sample sequence names for DX samples. External / sequence facility samples will return a key error and the original sample name will therefor not be overwritten. 

Maybe it is better to just test for sample within the dictionary sample_sequence_names?
Like so:
```python
if data[sample_name_index] in sample_sequence_names:
    data[sample_name_index] = sample_sequence_names[data[sample_name_index]]
```

Reply:
fixed

Reply:
:+1: